### PR TITLE
sdk: vaa: add missing chain IDs for gnosis and rootstock

### PR DESCRIPTION
**Pull request by nik-suri**

### CODE REVIEW

File: sdk/vaa/structs.go [hunk 1]
```diff
@@ -438,6 +440,8 @@ const (
 	ChainIDBase ChainID = 30
 	// ChainIDSei is the ChainID of Sei
 	ChainIDSei ChainID = 32
+	// ChainIDRootstock is the ChainID of Rootstock
+	ChainIDRootstock ChainID = 25
```
Comment (bruce-riley) [inline sdk/vaa/structs.go:444]: Rootstock should be 33.

**Comment by bruce-riley** [comment]
Also add to `sdk/vaa/structs_test.go`?

**Review by bruce-riley — DISMISSED** [review]

**Review by SEJeff — APPROVED** [review]

**Comment by SEJeff** [comment]
@nik-suri you prob want to update the protobuf definitions as well: https://github.com/wormhole-foundation/wormhole/blob/cd579d01003eeffcd8c67b83abd8587bf714fd24/node/pkg/proto/publicrpc/v1/publicrpc.pb.go#L27-L72

**Review by bruce-riley — APPROVED** [review]

**Review by SEJeff — APPROVED** [review]
Good spot on the pb!

**Review by bruce-riley — APPROVED** [review]

**Review by evan-gray — APPROVED** [review]

**Pull request merged**